### ISSUE TITLE
Bug: order value in component configuration not respected

### ISSUE THREAD

**Issue by paulrobertlloyd** [frctl/fractal#8]
I have the following components, with the `order` value set in their respective configs as follows:

```
a: 1
cite: 6
dfn: 8
abbr: 9
data: 10
code: 12
b: 19
br: 22
```

Yet, the components are displayed in the following (seemingly random) order in the UI:

```
a
b
br
cite
dfn
abbr
code
```

Here is an example config file:

```
label: "Abbreviation"
order: 9
notes: |
  The `abbr` element represents an abbreviation or acronym, optionally with its expansion. The `title` attribute may be used to provide an expansion of the abbreviation.
```

What’s going on? (This is just a subset of the components I’m trying to order, BTW).

**Comment by allmarkedup** [comment]
Yeah definitely looks like a bug... I'll look into this now.

**Comment by allmarkedup** [comment]
Fixed now in #42007719785bab2516fcfc810965547d620620d9 (& released in version v0.9.5)
